native/app/Source/Extensions/AudioDevice.swift

```diff
@@ -187,9 +187,15 @@ extension AudioDevice {
   
   static public var builtInOutputDevice: AudioDevice {
     get {
-      let device: AudioDevice? = AudioDevice.allOutputDevices().first( where: { (device) -> Bool in
+      var device: AudioDevice? = AudioDevice.allOutputDevices().first( where: { (device) -> Bool in
         device.transportType == TransportType.builtIn
       })
+      if device == nil {
+        sleep(4)
```

Comment:
Shouldn't use locking timers in the app unless absolutely necessary. Could you please message me on Discord and explain what are the cases when there's going to be no built-in output device? Is it during installation of the driver? If so, when we move the installation to .pkg file process will this still be an issue?

Reply:
Locking sleep was chosen as hotfix, because of all other logic around it that wasn't made to work in async way. But I will remove it now, since it was needed only when coreaudiod restarted and builtin was not ready yet.